arcgis-ios-sdk-samples/Geometry/Format coordinates/FormatCoordinates.storyboard

```diff
@@ -1,168 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13196" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="IIZ-g5-rn0">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="14460.31" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="IIZ-g5-rn0">
     <device id="retina4_7" orientation="portrait">
         <adaptation id="fullscreen"/>
     </device>
     <dependencies>
         <deployment identifier="iOS"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="13173"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14460.20"/>
         <capability name="Safe area layout guides" minToolsVersion="9.0"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <scenes>
         <!--Format Coordinates View Controller-->
         <scene sceneID="Tpr-Zl-3jf">
             <objects>
-                <viewController automaticallyAdjustsScrollViewInsets="NO" id="IIZ-g5-rn0" customClass="FormatCoordinatesViewController" customModule="arcgis_ios_sdk_samples" customModuleProvider="target" sceneMemberID="viewController">
+                <viewController automaticallyAdjustsScrollViewInsets="NO" id="IIZ-g5-rn0" customClass="FormatCoordinatesViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
                     <view key="view" contentMode="scaleToFill" id="nRX-gd-Kdt">
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="GzU-Me-xVd">
```

Comment:
I would suggest inverting the order of the subviews. It would make the whitespace at the bottom of the table view in regular height environments less noticeable.

Reply:
The stack view's leading and top constraints should be updated to use the superview rather than the safe area.

![screen shot 2018-12-03 at 10 41 10 am](https://user-images.githubusercontent.com/2257493/49394162-0914be00-f6e8-11e8-80e8-6014d1b85000.png)

Reply:
The issue with switching the order is that the text field area appears squished when the software keyboard is onscreen.

Reply:
I see what you mean. That makes sense.